src/reducers/tourReducer.js

```diff
@@ -0,0 +1,12 @@
+import {TOURS_FETCH_SUCCESS} from '../constants';
+
+export default (state = [], action) => {
```

Comment:
```js
const getInitialState = () => ({
    isLoading: false,
    error: null,
    data: []
});
```

Please use getInitialState instead of "state = []"